etc/emscripten/README.md

```diff
@@ -0,0 +1,19 @@
+Code to allow building gap to WASM using Emscripten.
+
+Files:
+
+- build.sh : Run as 'etc/emscripten/build.sh' from a fresh copy of GAP.
+
+Note that this built copy has one major weakness -- the 'gap.data' (which includes all packages files) is huge.
+
+
+- web-template : Uses 'xterm-pty' to create a "nice" interface to the Wasm GAP.
+
+See 'run-web-demo.sh' as an example on how to set up a working website.
+
+Note that this these scripts require a feature called "SharedArrayBuffer", which requires
+some webserver report.
```

Comment:
do you mean "support"?

Could you be a bit more specific what is needed? I googled a bit and I think it might be "Cross-origin isolation" ?